config/tokenizer/word-level-masked.yaml

```diff
@@ -0,0 +1,13 @@
+model:
+  _target_: tokenizers.models.WordLevel
+  unk_token: "<unk>"
+pre_tokenizer:
+  _target_: tokenizers.pre_tokenizers.Whitespace
+trainer:
```

Comment:
Can this be made more modular with word-level?  I.e. inheriting defaults from word-level, and then just adding "<MASK>" as a special token.  With these configs, we should try to "DRY", i.e. don't repeat yourself

Reply:
I'm having trouble defining the word-level-masked.yaml such that it inherits all the values (including trainer/special_tokens) from word-level.yaml, but then _adds_ a new value to trainer/special_tokens. I've looked at the hydra documentation, and haven't found something like an "extend" argument. Do you know of the appropriate hydra format? Or is it acceptable to inherit all the other values, and then just repeat the shared special_tokens?

Reply:
Yeah, I think you're right that Hydra doesn't let you extend a list provided from elsewhere / defaults.  But I do think it would be valuable to still inherit all the other values and repeat the shared special_tokens, as you suggest at the end there

Reply:
Ok, then it's done and pushed!